### ISSUE TITLE
Can't install add-ons, says they're corrupt

### ISSUE THREAD

**Issue by prochanunez17** [mozilla/addons#754]
### Describe the problem and steps to reproduce it:
The problem happens as soon as an add-on is supposed to be finished downloading.

### What happened?
I get an error message from a puzzle piece in the search bar that says "The add-on downloaded from this site could not be installed because it appears to be corrupt."
### What did you expect to happen?
The add-on should have installed, but instead I get an error.
### Anything else we should know?
![error](https://user-images.githubusercontent.com/33759938/42912809-46b8372c-8abf-11e8-9d02-8ca4d0aa68f9.png)
I'm using Firefox 57 on Xubuntu Linux LTS 18.04. Everything used to work fine.

**Comment by muffinresearch** [comment]
@prochanunez17 do you see the same behaviour with a new user profile?

**Comment by prochanunez17** [comment]
I was frustrated and decided to download Waterfox and use that instead, and everything worked fine with my profile on that. After returning to normal Firefox I found that all the add ons that I got on Waterfox were on Firefox, and I was able to download add ons again. I tried refreshing Firefox, reinstalling, and deleting some extension related files, but what worked was using Waterfox for a bit.
